src/validation.cpp

```diff
@@ -1107,15 +1106,10 @@ bool AcceptToMemoryPoolWorker(CTxMemPool& pool, CValidationState& state, const C
                 // unconfirmed ancestors anyway; doing otherwise is hopelessly
                 // insecure.
                 bool fReplacementOptOut = true;
-                if (fEnableReplacement)
-                {
-                    BOOST_FOREACH(const CTxIn &_txin, ptxConflicting->vin)
-                    {
-                        if (_txin.nSequence < std::numeric_limits<unsigned int>::max()-1)
-                        {
-                            fReplacementOptOut = false;
-                            break;
-                        }
+                BOOST_FOREACH(const CTxIn &_txin, ptxConflicting->vin) {
```

Comment:
this whole section is still about replacement, I think it should be removed.

As well as line 1115?

Reply:
Well, that's done on https://github.com/ElementsProject/elements/pull/249 I will rebase on top of this, just wanted to keep it separated in case we want the other parts but not this one.